tests/test_mixins.py

```diff
@@ -0,0 +1,68 @@
+import os
+from tests import SiteTest, manager
+
+class MixinTests(SiteTest):
+    """ Test cases for mixins """
+
+    def test_handler_5xx(self):
+        """ Check error view returns error message """
+        from werkzeug.exceptions import InternalServerError
+        from pysite.views.error_handlers import http_5xx
+
+        error_view = http_5xx.Error500View()
+        error_message = error_view.get(InternalServerError)
+        self.assertEqual(error_message[1], 500)
+
+    def test_route_view_runtime_error(self):
+        """ Check that wrong values for route view setup raises runtime error """
+        from pysite.base_route import RouteView
+
+        rv = RouteView()
+        try:
+            rv.setup(manager, 'sdfsdf')
+        except RuntimeError:
+            return True
+        raise Exception('Expected runtime error on setup() when giving wrongful arguments')
```

Comment:
this would probably also be nicer with `self.assertRaises`

Reply:
done! 👍